js/board.js

```diff
@@ -45,7 +45,7 @@ define(["js/properties", "js/sound", "js/squareImages", "js/info"], (properties,
     }
 
     function Board() {
-        this.squares = [];
+        this.squares = []
```

Comment:
I wasn't expecting a change here, if the ; got removed here this probably means that your eslint is not setup correctly.

Reply:
You're right I don't!
I have no idea why that is.
I'm not used to working with linting.

Reply:
What IDE are you using? If it's VS code you can just install the eslint plugin and it should lint your code automatically.